### PR TITLE
DF-1473 Event Sidebar

### PR DESCRIPTION
**Pull request by jimmynotjim**
Updated the event sidebar to include some demo content

## Additions

- None

## Removals

- None

## Changes

- Updated the object passed to `post_related.render()`
- Updated the list of related links

## Testing

Navigate to
- events/spring-2014-rulemaking-agenda/

## Review

- @anselmbradford 
- @sebworks 

## Preview

screen shot 2015-02-23 at 5 21 03 pm

## Notes

- Subscribe form can't currently support zip codes or event type
- Used existing Related Posts macro for code consistency

### CODE REVIEW

File: events/_single.html [hunk 1]
```diff
@@ -50,5 +49,18 @@ <h1 class="header-slug">
             {{ rss.render(vars.rss_path) }}
         </div>
     </section>
+    <section class="block">
+        {% import "related-links.html" as related_links %}
```
Comment (anselmbradford) [inline events/_single.html]: Should `related_links` be imported twice?

Comment (jimmynotjim) [inline events/_single.html]: Whoops, good catch.

**Comment by jimmynotjim** [comment]
If this is good I'll squash that fix in the morning so it's ready to merge before scrum.

**Comment by anselmbradford** [comment]
Yeah looks good otherwise! :+1:

**Comment by jimmynotjim** [comment]
Squashed, merge at will

**Pull request merged**